qa/cephfs/overrides/ignorelist_health.yaml

```diff
@@ -25,4 +25,6 @@ overrides:
       - BLUESTORE_SLOW_OP_ALERT
       - slow operation indications in BlueStore
       - experiencing slow operations in BlueStore
+      # cluster [WRN] Health check failed: 0 slow ops, oldest one blocked for 181 sec, osd.4 has slow ops (SLOW_OPS)
```

Comment:
nit: Did you mean to leave this here?

Reply:
I don't think we should have an op blocked that long. Please check the OSD logs to see what the op was and what was blocking it.

Reply:
@rishabh-d-dave anything on this?

Reply:
Haven't gotten the chance to re-visit this PR.